docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>As input convex neural networks (ICNN) are notoriously difficult to train (<a href="https://arxiv.org/pdf/2111.12187.pdf" target="_blank">Richter-Powell et al., 2021</a>), <a href="https://arxiv.org/pdf/2206.14262.pdf" target="_blank">Bunne et al. (2022)</a> propose to use closed-form solutions between Gaussian approximations to derive relevant parameter initializations for ICNNs: they propose, given two measures µ&nbsp;and ν, to initialize ICNN parameters so that they are (initially) meaningful in the context of OT, namely that its gradient is able to approximately map source measure&nbsp;µ&nbsp;onto&nbsp;target ν. These initializations rely on closed-form solutions available for Gaussian measures (<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/mana.19901470121" target="_blank">Gelbrich, 1990</a>). In this notebook, we introduce the <em>identity</em> and <em>Gaussian approximation</em>-based initialization schemes, and illustrate how they can be used within the <code>OTT</code> library and its <code>ICNN</code>-based <code>NeuralDual</code> module.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>